xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-bootswatch/src/main/resources/FlamingoThemes/Yeti.xml

```diff
@@ -2094,7 +2094,7 @@ input[type="checkbox"] {
   &gt; li:last-child {
     &gt; a,
     &gt; span {
-      border-radius: 3px;
+      border-radius: @border-radius-small;
```

Comment:
Same as above: since Color themes can decide their own border radiuses in the advanced section (or with variables), why are we changing the color themes here?